packages/cli/src/installers/node/index.ts

```diff
@@ -3,51 +3,25 @@ import { spawnSync } from "child_process"
 import chalk from "chalk"
 import inquirer from "inquirer"
 import { validatePushApiKey } from "@appsignal/core"
+import * as fs from "fs"
+import * as path from "path"
 
 import { SUPPORTED_NODEJS_INTEGRATIONS } from "../../constants"
 import { spawnDemo } from "../../commands/demo"
 
 /**
- * This is the very last thing to be displayed by the installer CLI! The
- * indendation is intentional, due to JavaScripts handling of multi-line
- * strings
+ * Installs the Node.js integration in the current working process.cwd().directory
```

Comment:
```suggestion
 * Installs the Node.js integration in the current working directory
```
I don't think you meant that to be there.